chapters/teams.Rmd

```diff
@@ -864,7 +864,7 @@ In a large project,
 a \gref{product manager}{product_manager} decides how important items are,\index{product manager}\index{project!product manager}
 while a \gref{project manager}{project_manager} is responsible for estimating effort\index{project manager}\index{project!project manager}
 and tracking progress.
-In a typical research software projects,
+In a typical research software project,
```

Comment:
Can also be 'In typical research software projects,', happy to change based on feedback.

Reply:
I think this looks good the way you've done it.